api/src/routes/alerts.rs

```diff
@@ -6,6 +6,19 @@ use mongodb::{
     bson::{Document, doc},
 };
 
+#[utoipa::path(
+    get,
+    path = "/alerts/{survey_name}/get_object/{object_id}",
```

Comment:
```suggestion
    path = "/alerts/{survey_name}/objects/{object_id}",
```

Might actually want this to be reworked as `GET /surveys/{survey_name}/alerts` and allow an `object_id` query parameter. I assume someone may want to fetch more alerts than just for a single object ID.

It might also be worth clarifying the relationship between a survey and a data catalog. Seems like a one-to-many relationship, i.e., a survey can have multiple catalogs associated with it.